tests/ed25519.rs

```diff
@@ -24,13 +24,13 @@ fn test_zip215() -> crypto::Result<()> {
     for tv in tvs.iter() {
         let mut pkb = [0; PUBLIC_KEY_LENGTH];
         hex::decode_to_slice(tv.public_key, &mut pkb as &mut [u8]).unwrap();
-        let pk = PublicKey::try_from_bytes(pkb)?;
+        let pk = PublicKeyBytes::from_bytes(pkb);
```

Comment:
These changes remove tests for `PublicKey`. Maybe, duplicate it like this:

```rust
let pk = PublicKey::try_from_bytes(pkb)?;
let pkb = PublicKeyBytes::from_bytes(pkb);
...
assert!(pk.verify(&sig, ms));
assert!(pkb.verify(&sib, ms));
```

Reply:
hm, ok